docs/user-tests/user-test-02-template-entwickler.md

```diff
@@ -0,0 +1,51 @@
+# User-Test für Entwickler
+
+**Tester:**
+
+**Datum:**
+
+## Aufgaben
+
+Folgende Aufgaben sollen im User-Test vorgenommen werden, getestet werden kann jedoch noch mehr. 
+Sofern daneben noch weitere Funktionen getestet wurden, würden wir Sie darum bitten, diese in der Tabelle zu vermerken.
+
+1) Bilder hinzufügen
```

Comment:
Frage mich, ob wir den Schritt davor, also die Einfachheit der Implementierung, irgendwie mitnehmen könnten?

Reply:
Meinst du mit "Einfachheit der Implementierung", dass der Code sofort verständlich sein soll?

Reply:
Also dieser Schritt unser Plugin in ein Softwareprojekt einzubauen.